tests/end-to-end/backup-restore-test/README.md

```diff
@@ -0,0 +1,103 @@
+# End-to-end Backup and Restore Tests
+
+## Overview
+
+This project contains end-to-end backup and restore tests for the Kyma installation. The tests run daily on Prow to validate if the backup and restore process works for all components.
+ 
+
+## Prerequisites
+
+Install [Ark](../../../resources/ark/README.md#details) and [configure](../../../docs/backup/docs/03-01-backup-configuration.md) it for a specific storage provider.
+
```

Comment:
Here should be also prerequisities about tools like go, docker etc., like here: https://github.com/kyma-project/kyma/tree/master/tests/end-to-end/upgrade#prerequisites (you can ctrl c, ctrl v 😄 )